PP_APP/server/frontend/templates/rate.html

```diff
@@ -4,12 +4,21 @@
 
 <section id="about" class="about-section bg-light">
     <div class="container">
+
+    <div class="alert alert-danger alert-dismissible collapse" role="alert">
+        <p id="status"></p>
+        <button type="button" class="close" onclick="$('.alert').hide()">
+            <span aria-hidden="true">&times;</span>
+        </button>
+    </div>
+
+    <img src="" id="res_img" class="rounded mx-auto d-block" style="max-height:0px;">
     <form id="imageUploadForm" method="post" enctype="multipart/form-data">
         {% csrf_token %}
-
+        
         <div class="row">
             <div class="col-sm">
-                <label id="top-dropzone" class="dropzone top-dropzone" style="background:linear-gradient(to bottom, rgba(22, 22, 22, 0.5) 0%, rgba(22, 22, 22, 0.8) 80%, #0000008a 100%), url({% static 'img/sample_top.jpg' %}) top left no-repeat;background-size:cover;">                    
+                <label id="top-dropzone" class="dropzone top-dropzone" style="background: url({% static 'img/sample_top.png' %}) top left no-repeat;background-size:cover;">                    
```

Comment:
The white background and the white text are really difficult to distinguish. You probably edited these images yourself, maybe try giving them a darker background?